scripts/spec/uploader_spec.rb

```diff
@@ -2,7 +2,7 @@
 require 'uploader'
 
 describe Build::Uploader do
-  subject { described_class.new("directory", "type") }
+  subject { described_class.new("directory", "type", "delete") }
```

Comment:
Shouldn't `"delete"` be a boolean instead?

Reply:
You're right. That's not used in the spec, but should match. Fixed.

Reply:
Thanks!